docs/reference/announcements/880.md

```diff
@@ -123,25 +123,20 @@ Additional upgrade considerations are necessary for deployments that use custom
 
 ## Camunda Exporter <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
 
-A new Camunda Exporter brings the importer and archiving logic of web components (Tasklist and Operate) closer to the distributed platform (Zeebe). The index schema is also being harmonized.
+Our web applications have historically used importers and archivers to consume, aggregate, and archive historical data provided by our Elasticsearch (ES) or OpenSearch (OS) Exporter.
 
-The exporter can consume Zeebe records (mostly events created by the engine), aggregate data, and store the related data into shared and harmonized indices.
+![current-state](../img/current-state-refined.png)
 
-- Data is archived in the background, coupled to the exporter but without blocking the exporter's progress.
-- Indices can be located in either Elasticsearch (ES) or OpenSearch (OS). Our web components (Tasklist and Operate) then use the new harmonized indices to show data to the user.
+In the past release, we’ve engineered a new Camunda Exporter, which brings the importer and archiving logic of web components (Tasklist and Operate) closer to our distributed platform (Zeebe). This allows us to simplify our installation, enable scalability for web apps, reduce the latency to show runtime and historical data, and reduce data duplication (resource consumption).
 
-The following diagram shows a simplified version of this work.
+![target](../img/target-brown-field-refined-without-optimize.png)
```

Comment:
I'd suggest using `green/brown` for terminology (only `brown` is shown in the PR).